Customize Ansible cfg persistently (#1929)
theforeman/foreman-documentation: guides/common/modules/ref_customizing-ansible-configuration.adoc

[id="customizing-ansible-configuration_{context}"]
= Customizing Ansible Configuration

{Project} manages essential Ansible configuration that is required for Ansible integration with {Project}.
However, you can customize other Ansible configuration options as usual.

{Project} stores the essential Ansible configuration as environment variables in `/etc/foreman-proxy/ansible.env`.

> This file is refreshed on each run of {foreman-installer}.

Reviewer: ```suggestion
This file is managed by the {foreman-installer}.
```
